### PR TITLE
fix null ref exception

### PR DESCRIPTION
**Pull request by stishkin**

### CODE REVIEW

File: src/ApiService/ApiService/Log.cs [hunk 1]
```diff
@@ -25,7 +25,9 @@ public struct LogStringHandler {
     }
 
     public void AppendFormatted<T>(T message) {
-        _builder.Append(message?.ToString());
+        if (message is not null) {
+            _builder.Append(message.ToString());
```
Comment (Porges) [inline src/ApiService/ApiService/Log.cs:29]: do we want to put a placeholder in like `<null>`?

File: src/ApiService/ApiService/Log.cs [hunk 2]
```diff
@@ -105,14 +109,12 @@ class AppInsights : ILog {
...
 
-
         Dictionary<string, double>? copyMetrics = null;
         if (metrics is not null) {
             copyMetrics = new(metrics);
         }
         _telemetryClient.TrackException(ex, copyTags, copyMetrics);
-
-        Log(correlationId, $"{message} : {ex.Message}", SeverityLevel.Error, tags, caller);
+        Log(correlationId, $"[{message}] {ex.Message}", SeverityLevel.Error, tags, caller);
```
Comment (Porges) [inline src/ApiService/ApiService/Log.cs:117]: the callstack made me wonder if it was `ex` that was `null` here

Comment (stishkin) [reply src/ApiService/ApiService/Log.cs:117]: it's string builder inside of message

I reproed

Comment (stishkin) [reply src/ApiService/ApiService/Log.cs:117]: (and then I'll be proven wrong ....)

**Review by Porges — APPROVED** [review]

**Pull request merged**